web/modules/custom/vote_utility/translations/am.po

```diff
@@ -95,6 +95,9 @@ msgstr "የመራጮች ምዝገባ የመጨረሻ ቀናት"
 msgid "In person registration deadline:"
 msgstr "በአካል ምዝገባ የመጨረሻ ቀን፦"
 
```

Comment:
The following was added using Google Translate as we did not have a translation for it.